pkgs/applications/misc/mu-repo/default.nix

```
@@ -0,0 +1,25 @@
+{ lib, fetchFromGitHub, buildPythonApplication, pytest, git }:
+
+buildPythonApplication rec {
+  pname = "mu-repo";
+  version = "1.8.0";
+
+  src = fetchFromGitHub {
+    owner = "fabioz";
+    repo = pname;
+    rev = with lib;
+      "mu_repo_" + concatStringsSep "_" (splitVersion version);
+    sha256 = "1dxfggzbhiips0ww2s93yba9842ycp0i3x2i8vvcx0vgicv3rv6f";
+  };
+
+  checkInputs = [ pytest git ];
+  checkPhase = "py.test mu_repo";
```

to fix darwin tests, the fetchFromGitHub pulls a tar ball, so it doesn't make sense to do this test.
```suggestion
  checkPhase = "py.test mu_repo --ignore=mu_repo/tests/test_checkout.py";
```

---

Fixed.